message-campaign/src/test/java/org/motechproject/messagecampaign/it/MessageCampaignServiceBundleIT.java

```diff
@@ -63,7 +63,7 @@ public void setup() {
     public void shouldUnscheduleMessageJobsWhenCampaignIsStopped() throws SchedulerException {
         CampaignRequest campaignRequest = new CampaignRequest("entity_1", "PREGNANCY", new LocalDate(2020, 7, 10), null);
 
-        TriggerKey triggerKey = triggerKey("org.motechproject.messagecampaign.fired-campaign-message-MessageJob.PREGNANCY.entity_1.PREGNANCY", "default");
+        TriggerKey triggerKey = triggerKey("org.motechproject.messagecampaign.fired-campaign-message-MessageJob.PREGNANCY.entity_1.PREGNANCY-dayofweek", "default");
```

Comment:
Can we also have tests for no prefix version? It's essential that campaigns created in the past still work after this change.